server/index.js

```diff
@@ -0,0 +1,67 @@
+const { PrismaClient } = require('@prisma/client');
+
+const express = require('express');
+const bodyParser = require('body-parser');
+const bcrypt = require('bcrypt');
+
+const prisma = new PrismaClient();
+const saltRounds = 10;
+const app = express();
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.json());
+
+app.post('/api/user/signup', async (req, res) => {
```

Comment:
This route wasn't tested. It is always good to test using the TDD approach. Try to look into Jest

Reply:
@chadwalt  Please review. I have added tests for the signup endpoint

Reply:
I also think defining a routes into their own file, would help with testing, clean structuring.